hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/WriteOperations.java

```diff
@@ -192,11 +192,10 @@ int abortMultipartUploadsUnderPath(String prefix)
       throws IOException;
 
   /**
-   * Abort multipart uploads under a path: limited to the first
-   * few hundred.
-   * @param prefix prefix for uploads to abort
-   * @return a count of aborts
-   * @throws IOException trouble; FileNotFoundExceptions are swallowed.
+   * List in-progress multipart uploads under a path
```

Comment:
does still look limited to a few hundred, so keep that bit in, as well as adding a "." at the end. see the s3afs method it calls....

Reply:
Sure, I updated.